db/schema.rb

```diff
@@ -130,6 +130,17 @@
     t.index ["content_id", "locale", "with_drafts"], name: "expanded_links_content_id_locale_with_drafts_index", unique: true
   end
 
+  create_table "link_changes", force: :cascade do |t|
+    t.uuid "source_content_id", null: false
+    t.uuid "target_content_id", null: false
+    t.string "link_type", null: false
+    t.integer "change", null: false
```

Comment:
I'm not sure an integer is the right field here since we only care about two values. Maybe it's better to have an enum (well the rails pseudo enum) assuming bool is a bit too crude.